gluten-core/src/main/scala/org/apache/spark/sql/execution/ColumnarShuffleExchangeExec.scala

```diff
@@ -163,6 +163,8 @@ case class ColumnarShuffleExchangeExec(
 
   protected def withNewChildInternal(newChild: SparkPlan): ColumnarShuffleExchangeExec =
     copy(child = newChild)
+
+  def advisoryPartitionSize: Option[Long] = None
```

Comment:
shall we add it to constructor to respect `advisoryPartitionSize` ?

Reply:
Sure. Updated.